WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1530,6 +1530,19 @@ local methods = {
     -- no addon, or no data, or ignore flag
     return false, false, nil, nil
   end,
+  ["RefreshBT2UpgradeIcon"] = function(self)
+    if not self.data.controlledChildren and self.data.triggers then
+      for i, t in pairs(self.data.triggers) do
```

Comment:
This is such a small nitpick, that I feel bad about point it out. But this should be ipairs.